dist/svg.select.js

```diff
@@ -375,8 +405,10 @@ SVG.Element.prototype.selectize.defaults = {
     pointsExclude: [],                       // easier option if to exclude few than rewrite all
     classRect: 'svg_select_boundingRect',    // Css-class added to the rect
     classPoints: 'svg_select_points',        // Css-class added to the points
-    radius: 7,                               // radius of the points
+    radius: 7,                               // radius of the points, only for pointType: 'circle'
+    rectSize: 8,                             // size of rect points, only for pointType: 'rect'
```

Comment:
I feel like both option  should be unified under one name. Maybe pointSize.

Reply:
I've added the `pointSize` option and dropped the `radius` option. I guess this should be major version now, since it wouldn't be compatible with previous options. I can leave radius if you wish.